### ISSUE TITLE
incorrect `hit` count in status bar

### ISSUE THREAD

**Issue by deargle** [NYUCCL/psiTurk#351]
when a hit is created, it is not immediately available via a boto call to list hits -- it takes a few seconds.  Because of the delay, the `hit` count in the cmd status bar is one lower than it should be, and stays this way until the next internal shell call to `update hit tally`.

Possible solutions:
* call `update_hit_tally` after each command (expensive!)
* track more than just the hit_id in the hits table, and keep this table up to date for each HIT modification call (deletion, extension, etc). Have the tally be dependent on the local table count instead of an mturk call. <-- long-term goal

**Comment by ariekahn** [comment]
If we're just talking a few seconds, it make any sense to just pause the shell for a few seconds, until the query includes the new HIT? Could display a message like 'waiting to verify HIT' and time out if nothing happens after, say, 10 seconds.

**Comment by deargle** [comment]
Yeah, I was thinking a wait too. I was also considering just artificially bumping up the hit tally by one, which would be corrected the next time update_hit_tally were called. The downside of this is that an immediate call to `hit_list` would display one less than expected. Will probably just do delay.
